src/desktop/apps/auction/reducers/__tests__/artworkBrowser.test.js

```diff
@@ -153,6 +153,22 @@ describe("auction/actions/artworkBrowser.test.js", () => {
         ])
         notAllArtworksFetched.artworkBrowser.total.should.eql(10)
       })
+
+      it("updates allFetched to true if the current page exceeds 100", () => {
+        const maxPageResponse = auctions(
```

Comment:
This starts off our initial state with page 100, and a total that doesn't match the fetched artworks (which is what triggers a further fetch).